vip-helpers/vip-utils.php

```diff
@@ -771,24 +775,26 @@ function vip_safe_wp_remote_request( $url, $fallback_value='', $threshold=3, $ti
 
 	$elapsed = ( $end - $start ) > $timeout;
 	if ( true === $elapsed ) {
-		if ( false !== $option && $option['hits'] < $threshold )
+		if ( false !== $option && $option['hits'] < $threshold ) {
 			wp_cache_set( $cache_key, array( 'time' => floor( $end ), 'hits' => $option['hits']+1 ), $cache_group, $retry );
-		else if ( false !== $option && $option['hits'] == $threshold )
+		} else if ( false !== $option && $option['hits'] == $threshold ) {
 			wp_cache_set( $cache_key, array( 'time' => floor( $end ), 'hits' => $threshold ), $cache_group, $retry );
-		else
+		} else {
 			wp_cache_set( $cache_key, array( 'time' => floor( $end ), 'hits' => 1 ), $cache_group, $retry );
+		}
 	}
 	else {
-		if ( false !== $option && $option['hits'] > 0 && time() - $option['time'] < $retry )
+		if ( false !== $option && $option['hits'] > 0 && time() - $option['time'] < $retry ) {
 			wp_cache_set( $cache_key, array( 'time' => $option['time'], 'hits' => $option['hits']-1 ), $cache_group, $retry );
-		else
+		} else {
 			wp_cache_delete( $cache_key, $cache_group);
+		}
 	}
 
 	if ( is_wp_error( $response ) ) {
 		// Log errors for internal WP.com debugging
 		if ( ! defined( 'WPCOM_VIP_DISABLE_REMOTE_REQUEST_ERROR_REPORTING' ) || ! WPCOM_VIP_DISABLE_REMOTE_REQUEST_ERROR_REPORTING ) {
-			error_log( "vip_safe_wp_remote_request: Blog ID {$blog_id}: Requesting $url with a timeout of $timeout failed. Result: " . maybe_serialize( $response ) );
+			trigger_error( "vip_safe_wp_remote_request: Blog ID {$blog_id}: Requesting $url with method {$parsed_args[ 'method' ]} and a timeout of $timeout failed. Result: " . maybe_serialize( $response ), E_USER_NOTICE );
```

Comment:
I think by default, WordPress force the error reporting level to ignore notices so I don't think these will actually make it to Logstash.

We will probably want to use `E_USER_WARNING` instead.

```suggestion
			trigger_error( "vip_safe_wp_remote_request: Blog ID {$blog_id}: Requesting $url with method {$parsed_args[ 'method' ]} and a timeout of $timeout failed. Result: " . maybe_serialize( $response ), E_USER_WARNING );
```

Reply:
I checked Logstash and found lots of Notices so they do make it, but a warning might be better anyway?

Reply:
👍 1e0e8ad

Reply:
On second thought, I'm not sure `Warning` is the right level for this. It could go either way, really, I just know we'll cause an increase in global warnings with this change, when there's not really anything actionable (which is more of a notice).